stanford/roles/cluster/defaults/main.yml

```diff
@@ -230,14 +230,14 @@ CLUSTER_ELBS:
   forum:
     - name: "{{ COMMON_DEPLOYMENT }}-elb-forum-private"
       health_check:
-        ping_protocol: tcp
-        ping_port: 80
```

Comment:
don't we need the `ping_port`?

Reply:
I think it's optional/implied due to `http`, but we do include it explicitly for the other ELBs, so I'll add it back.